.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
+1 for adding prettier config
but it seems like not all files are properly formatted
should we add prettier to the dev dependencies too?

Reply:
Just pushed an update doing just that + an npm script to run across the repo